packages/driver/src/cypress.ts

```diff
@@ -613,13 +613,25 @@ class $Cypress {
       case 'command:log:added':
         this.runner?.addLog(args[0], this.config('isInteractive'))
```

Comment:
Shouldn't this occur after the hidden check so the `runner` doesn't even get notified of the log? Could also add a new `_command:log:added` action that would emit the `_log:added` event.

Reply:
[`ccbeea0` (#28531)](https://github.com/cypress-io/cypress/pull/28531/commits/ccbeea0bfaf24861ebb0d38c53ee6cba94bbddd2)